lib/elements/satin_column.py

```diff
@@ -1093,20 +1093,28 @@ def plot_points_on_rails(self, spacing, offset_px=(0, 0), offset_proportional=(0
 
     def do_contour_underlay(self):
         # "contour walk" underlay: do stitches up one side and down the
-        # other.
+        # other. if the two sides are far away, adding a running stitch to travel
+        # in between avoids a long jump or a trim.
+
         pairs = self.plot_points_on_rails(
             self.contour_underlay_stitch_length,
             -self.contour_underlay_inset_px, -self.contour_underlay_inset_percent/100)
 
         if self._center_walk_is_odd():
-            stitches = [p[0] for p in reversed(pairs)] + [p[1] for p in pairs]
+            first_side = [p[0] for p in reversed(pairs)]
+            second_side = [p[1] for p in pairs]
         else:
-            stitches = [p[1] for p in pairs] + [p[0] for p in reversed(pairs)]
+            first_side = [p[1] for p in pairs]
+            second_side = [p[0] for p in reversed(pairs)]
 
-        return StitchGroup(
+        patch = StitchGroup(
```

Comment:
I've been trying to change from using the word "patch" over to "stitch group".  Would you mind changing this variable to `stitch_group`?  "Patch" is confusing because it sounds like an embroidered patch, that's why I've been trying to change it.